specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json

```diff
@@ -1014,12 +1055,76 @@
     }
   },
   "definitions": {
+    "ImportImageParameters": {
+      "required": [
+        "sourceRepository",
+        "source"
+      ],
+      "type": "object",
+      "properties": {
+        "sourceRepository": {
+          "description": "Repository name of the source image.",
+          "type": "string"
+        },
+        "sourceTag": {
+          "description": "The tag name of the source image.  When both source tag and source manifest are omitted the 'latest' tag will be used.\r\nExclusive with SourceManifestDigest.",
+          "type": "string"
+        },
+        "sourceManifestDigest": {
+          "description": "The manifest sha of the source image. Exclusive with SourceTag.",
+          "type": "string"
+        },
```

Comment:
I wonder if `sourceRepository`, `sourceTag`, and `sourceManifestDigest` can be combined to one single parameter such as `sourceImage` which can be the following format:
 - `hello-world` -> `hello-world:latest`.
 - `hello-world:v21
 - `hello-world@sha256:abc123`

This will be consistent with `docker` and avoid the mentioning of `digest` is exclusive with `tag`.
/cc @sajayantony

Reply:
made an update to this and updated the samples